openapi/Swarm.yaml

```diff
@@ -861,11 +861,107 @@ paths:
               schema:
                 $ref: 'SwarmCommon.yaml#/components/schemas/ReferenceResponse'
         '400':
-           $ref: 'SwarmCommon.yaml#/components/responses/400'
+          $ref: 'SwarmCommon.yaml#/components/responses/400'
         '401':
-           $ref: 'SwarmCommon.yaml#/components/responses/401'
+          $ref: 'SwarmCommon.yaml#/components/responses/401'
         '500':
            $ref: 'SwarmCommon.yaml#/components/responses/500'
         default:
           description: Default response
 
+  '/feed/{owner}/{topic}':
```

Comment:
In `router.go` the endpoint is called `/feeds`

https://github.com/ethersphere/bee/blob/aa61c6532d8b7c57c5776250c62264e333feef35/pkg/api/router.go#L92